src/typed-method-types/workflows/triggers/event-data/app_mentioned.ts

```diff
@@ -0,0 +1,34 @@
+export const AppMentioned = {
```

Comment:
Do we need to have a shared set of the common fields (e.g. the fields that are above `data`)? Every event should have these common fields, for example: 

```
	"team_id" => \encoded_team_id_t,
	"enterprise_id" => ?\encoded_team_id_t,
	"event_id" => string,
	"event_timestamp" => int,
	"type" => TriggerTypeApi,
	"data" => dict[]
);
```

Copy/pasting them into every one of these events doesn't feel right, though-- is there a way we can have these 'extend' something that contains the common constants? Or if not, maybe add a new `TriggerData.Event.common_fields` constant?

Reply:
Ooooh, good callout Jake! These are just referenced via the same templatized string, right? If so, would a pivot from `TriggerContextData` to `TriggerContext` work? For example

```ts
export const TriggerContext = {
  TeamId: "{{team_id}}",
  EnterpriseId: "{{enterprise_id}}",
  EventId: "{{event_id}}",
  EventTimestamp: "{{event_timestamp}}",
  Type: "{{type}}",
  Data: TriggerContextData
}
```
This would make all these reference strings available, and the data field is just accessed via `TriggerContext.Data`

Reply:
I'll tackle this in a separate PR!